pic-sure-hpds-ui/src/main/webapp/picsureui/common/router.js

```diff
@@ -108,6 +110,20 @@ define([
             footerView.render();
             $('#footer-content').html(footerView.$el);
         },
+        shouldDisplayBanner: function (config) {
+            let currentDate = moment().tz("America/New_York").toDate();
+            // Convert the start and end dates to UTC -5 (EST).
+            // They are string values, so we need to convert them to Date objects.
+            // The format of the string is "YYYY-MM-DD HH:mm:ss"
+            let startDate = moment(config.startDate).tz("America/New_York").toDate();
+            let endDate = moment(config.endDate).tz("America/New_York").toDate();
+
+            return (
+                currentDate >= startDate &&
+                currentDate <= endDate &&
+                config.styles && config.text && config.disabled !== true
```

Comment:
If you're allowing the use of a class, there may be instances where style is not defined. Probably, the most valuable logic is the text and disabled-- you might not have to check if style or class are set.

Reply:
I will do this before the follow up review.